PROFILING.md

```diff
@@ -0,0 +1,99 @@
+# Profiling Besu with `asprof`
+
+This guide explains how to set up and run [Async Profiler](https://github.com/jvm-profiling-tools/async-profiler) to profile a running Besu instance.
+
+---
+
+## 1. Setup
+
+### a. Download and Extract
+
+Download version 4.0 of Async Profiler appropriate for your architecture:
```

Comment:
We need to refer to Async profiler documentation to download the last release.

Reply:
I've added a link to their release page and left the wget command for version 4.0 as an example